Frontend/android-native/app/src/main/res/layout/floating_widget_layout.xml

```diff
@@ -44,7 +44,7 @@
                     android:layout_height="64dp"
                     android:background="@drawable/silent_popup_background"
                     android:orientation="vertical"
-                    android:visibility="gone"
+                    android:visibility="visible"
```

Comment:
@Anuragini-2506 this we are already changing it to visible dynamically already right? Then why is this required?